apps/studio/components/layouts/ProjectLayout/ProjectLayout.tsx

```diff
@@ -149,7 +149,7 @@ const ProjectLayout = ({
             />
             <ResizablePanel id="panel-right" className="h-full flex flex-col">
               {!navLayoutV2 && !hideHeader && IS_PLATFORM && <LayoutHeader />}
-              <main className="h-full flex flex-col flex-1 w-full overflow-x-hidden">
+              <main className="h-screen flex flex-col flex-1 w-full overflow-x-hidden">
```

Comment:
this is probably okay but just to check what this fixes?

Reply:
ah, lemme revert, this was me trying to fix a height issue